### PR TITLE
PJ_SIM_SUNBNB_2023-45 [BE] Vuetify and Router

### PR DESCRIPTION
**Pull request by andreenriquez11**
# Issue Link
https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-4

# Definition of Done
* [x] Update consolidate package
* [x] Install Vuetify 
* [x] Manually install router
* [x] Add pages folder

# Notes
n/a

# Screenshots/Recordings
n/a

### CODE REVIEW

File: fe/.env.example
Comment (LennardUy104) [inline fe/.env.example:1]: Do not delete .env.example since we are using this to the secret keys of the application

**Review by LennardUy104 — CHANGES_REQUESTED** [review]
Please restore .env.example file

File: fe/.env.example
Comment (andreenriquez11) [reply fe/.env.example:1]: Restored already @LennardUy104

Comment (LennardUy104) [reply fe/.env.example:1]: Nice! thanks

**Review by roseaugusto — APPROVED** [review]

**Pull request merged**